### ISSUE TITLE
Binary builds failing: missing 'jaraco' package

### ISSUE THREAD

**Issue by emlys** [natcap/invest#842]
Windows and mac binary builds failed on `main` with this error:
```
dist/invest/invest list
Traceback (most recent call last):
  File "Lib\site-packages\PyInstaller\hooks\rthooks\pyi_rth_pkgres.py", line 16, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "pkg_resources\__init__.py", line 74, in <module>
  File "pkg_resources\extern\__init__.py", line 52, in create_module
  File "pkg_resources\extern\__init__.py", line 44, in load_module
ImportError: The 'jaraco' package is required; normally this is bundled with this package so if you get this warning, consult the packager of your distribution.
[4832] Failed to execute script 'pyi_rth_pkgres' due to unhandled exception!
```

**Comment by dcdenu4** [comment]
Huh, looking at a build that failed (https://github.com/natcap/invest/runs/5041610588?check_suite_focus=true) and one that succeeded (https://github.com/natcap/invest/runs/5030480651?check_suite_focus=true), they both are using Pyinstaller 4.8. I also only see the Mac Binaries failing, but I'm just looking in the invest actions list https://github.com/natcap/invest/actions.

**Comment by emlys** [comment]
Looks like it's a combo of Pyinstaller and setuptools 60.7.0 which was released 10 hours ago: https://github.com/pyinstaller/pyinstaller/issues/6565

**Comment by emlys** [comment]
Here's where both binary builds failed: https://github.com/natcap/invest/runs/5042425168?check_suite_focus=true

**Comment by dcdenu4** [comment]
Nice, seems like they are quickly working on a patch.

**Comment by emlys** [comment]
Pyinstaller handles this now, so we can remove the setuptools pin and instead require pyinstaller>=4.9.

**Comment by emlys** [comment]
This was fixed by pinning pyinstaller >=4.10 in https://github.com/natcap/invest/pull/917.
